distribution/lib/Standard/Test/0.0.0-dev/src/Bench.enso

```diff
@@ -1,18 +1,33 @@
 from Standard.Base import all
 import Standard.Base.Runtime.Ref.Ref
 
-
+## Configuration for a measurement phase or a warmup phase of a benchmark.
+type Phase_Conf
+    ## The minimal number of seconds for which the phase should run.
+    Secs (secs:Integer)
+    ## Number of iterations of the phase.
+    Iters (iters:Integer)
+
+    to_text self = case self of
+        Phase_Conf.Secs secs -> secs.to_text + " seconds"
+        Phase_Conf.Iters iters -> iters.to_text + " iterations"
```

Comment:
Add conversion:
```ruby
Phase_Conf.from (that:Integer) = Phase_Conf.Iters that
```
just to make it simpler to `. set_warmup 3 . set_measure 3` and to show what Enso can do.